pkgs/development/python-modules/uproot/default.nix

```diff
@@ -0,0 +1,27 @@
+{lib, fetchFromGitHub, buildPythonPackage, numpy }:
+
+buildPythonPackage rec {
+  pname = "uproot";
+  version = "2.8.16";
+  src = fetchFromGitHub {
```

Comment:
Please use `fetchPypi` instead: https://pypi.org/project/uproot/. This allows easier automatic updating.